samples/eShopLite/AppHost/mymanifest.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@davidfowl Here's the new `manifest.json`. I don't have docker on my machine so the manifest is only for the front end app and the http service. I will redo the program.cs change before the merge.